google-scripts/process-form-submission/main.ts

```diff
@@ -18,17 +18,19 @@ export function onFormSubmit(
       `${REFERRALS_SHEET_NAME}`
     );
 
-    const currentUniqueId = setUniqueIdOnSubmission(
+    const currentUniqueSubmissionDetails = setUniqueIdOnSubmission(
```

Comment:
Maybe we could rename this to `currentSubmissionDetails` i.e without unique.

Reply:
Done in cf5cccc